.idea/inspectionProfiles/profiles_settings.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
The opening and closing tags of the component should be on separate lines for better readability.